pkg/web/healthzHandler.go

```diff
@@ -32,5 +35,18 @@ func (hh *healthHandler) Healthz(w http.ResponseWriter, r *http.Request) {
 }
 
 func (hh *healthHandler) Ping(w http.ResponseWriter, r *http.Request) {
+	err := hh.pingTarget.Ping()
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
Let's not return details about internal server errors. It's generally accepted to be a bad security practice.

The metrics service is actually using a library I wrote called [boom](https://github.com/darahayes/go-boom) in other places. I don't have a huge opinion on this but perhaps we could use it here instead.

example usage:

```golang
if err != nil {
  boom.badImplementation(w)
}
```

This will return a 500 error and a JSON payload with a generic error message.

Either that or simply change it to something like this:

```go
http.Error(w, "Internal Server Error", http.StatusInternalServerError)
```

Reply:
Will do, came into this issue assuming that the implementation for the healthz endpoint with `http.Error` was what you guys were looking for, but I see it was just a stub :+1: